ra/registration-authority.go

```diff
@@ -98,6 +98,7 @@ func NewRegistrationAuthorityImpl(clk clock.Clock, logger *blog.AuditLogger, sta
 const (
 	unparseableEmailDetail = "not a valid e-mail address"
 	emptyDNSResponseDetail = "empty DNS response"
+	multipleAddressDetail  = "multiple e-mail addresses parsed from single contact field"
```

Comment:
This would probably be clearer as "more than one @ symbol in email address."

Thanks for picking up this bug!